ods_ci/tests/Tests/1000__model_serving/1009__model_serving_triton_on_kserve/1009__model_serving_triton_on_kserve.robot

```diff
@@ -50,10 +50,17 @@ ${PYTHON_MODEL_NAME}=    python
 ${EXPECTED_INFERENCE_PYTHON_REST_OUTPUT_FILE}=      tests/Resources/Files/triton/kserve-triton-python-rest-output.json
 ${INFERENCE_REST_INPUT_PYTHON}=    @tests/Resources/Files/triton/kserve-triton-python-rest-input.json
 ${KERAS_RUNTIME_NAME}=    triton-keras-rest
-${KERAS_MODEL_NAME}=      resnet50
 ${KERAS_RUNTIME_FILEPATH}=    ${RESOURCES_DIRPATH}/triton_keras_rest_servingruntime.yaml
 ${INFERENCE_REST_INPUT_KERAS}=    @tests/Resources/Files/triton/kserve-triton-keras-rest-input.json
```

Comment:
this file is being deleted in this PR. You sure?

Reply:
yes

Reply:
then if you don't need to use it anymore, pls delete also this variable no?

Reply:
We need it for other tasks